blog/config/routes.rb

```diff
@@ -0,0 +1,68 @@
+Rails.application.routes.draw do
+  get 'welcome/index'
+
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+
+  # You can have the root of your site routed with "root"
+  # root 'welcome#index'
+
+  Rails.application.routes.draw do
```

Comment:
shouldn't draw routes inside of routes. think this is just an error and lines 10-13 should be removed.